Jantimizer/Histograms/HistogramModels/HistogramBucket.cs

```diff
@@ -3,17 +3,19 @@
     public class HistogramBucket : IHistogramBucket
     {
         public int ValueStart { get; }
+        public int ValueEnd { get; set; }
         public int Count { get; set; }
 
-        public HistogramBucket(int valueStart, int count)
+        public HistogramBucket(int valueStart, int valueEnd, int count)
         {
             ValueStart = valueStart;
+            ValueEnd = valueEnd;
             Count = count;
         }
 
         public override string? ToString()
         {
-            return $"Start: [{ValueStart}], Count: [{Count}]";
+            return $"Start: [{ValueStart}], End: |{ValueEnd}|, Count: [{Count}]";
```

Comment:
Is there a reason for the value of "End" is encapsulated differently from everything else?

Reply:
Maybe use the mathematical range notation [start, end] ?